dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -949,6 +950,72 @@ def run_generate_constraints_in_parallel(
     )
 
 
+@release_management.command(
+    name="tag-providers",
+    help="Generates tags for airflow provider releases.",
+)
+@option_dry_run
+@option_verbose
+def tag_providers():
+    found_remote = None
+    remotes = ["origin", "apache"]
+    for remote in remotes:
+        try:
+            command = ["git", "remote", "get-url", "--push", shlex.quote(remote)]
+            result = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.DEVNULL, text=True)
+            if "apache/airflow.git" in result.stdout:
+                found_remote = remote
+                break
+        except subprocess.CalledProcessError:
+            pass
+
+    if found_remote is None:
+        raise ValueError("Could not find remote configured to push to apache/airflow")
+
+    tags = []
+    for file in os.listdir(os.path.join(SOURCE_DIR_PATH, "dist")):
+        if file.endswith(".whl"):
+            match = re.match(r".*airflow_providers_(.*)-(.*)-py3.*", file)
+            if match:
+                provider = f"providers-{match.group(1).replace('_', '-')}"
+                tag = f"{provider}/{match.group(2)}"
+                try:
+                    subprocess.run(
```

Comment:
you should use `run_command` for those from `breeze` utils. This one wraps teh `subprocess.run` and automatically prints what's going on (including built-in support for `--dry-run` and `--verbose` commands. See how it's done in other commands.